docs/docs/python-sdk/guides/python-typing.mdx

```diff
@@ -101,3 +101,51 @@ my_object = client.get(MyOwnObject, name__value="example")
 ```
 
 > if you don't have your own Python module, it's possible to use relative path by having the `procotols.py` in the same directory as your script/transform/generator
+
+## Generating Pydantic models from GraphQL queries
```

Comment:
As you're working with this file can you also fix the typo with `procotols.py` just above that code rabbit is complaining about?